src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/model/OffenderAliasPendingDeletion.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.hmpps.nomis.datacompliance.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.time.LocalDate;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "OFFENDERS")
+public class OffenderAliasPendingDeletion {
+
+    @Id
+    @Column(name = "OFFENDER_ID", nullable = false)
+    private Long offenderId;
+
+    @Column(name = "OFFENDER_ID_DISPLAY", nullable = false)
+    private String offenderNumber;
+
+    @Column(name = "FIRST_NAME")
+    private String firstName;
+
+    @Column(name = "MIDDLE_NAME")
+    private String middleName;
+
+    @Column(name = "LAST_NAME", nullable = false)
+    private String lastName;
+
+    @Column(name = "BIRTH_DATE")
+    private LocalDate birthDate;
```

Comment:
would dateOfBirth be a better name

Reply:
I've just been mirroring the names in NOMIS to try to avoid confusion